src/EDF.jl

```diff
@@ -168,39 +168,20 @@ Type representing the header record for an `AnnotationList`
 # Fields
 
 * `n_samples` (`Int16`): The number of samples in a single data record
+* `offset_in_file` (`Int`): The annotation header's position,
+   relative to other signals in its origin file
 """
 struct AnnotationListHeader
     n_samples::Int16
+    offset_in_file::Int
 end
 
-AnnotationListHeader(header::SignalHeader) = AnnotationListHeader(header.n_samples)
+AnnotationListHeader(header::SignalHeader, offset::Int) = AnnotationListHeader(header.n_samples, offset)
```

Comment:
```suggestion
function AnnotationListHeader(header::SignalHeader, offset::Integer)
    return AnnotationListHeader(header.n_samples, Int(offset))
end
```

Reply:
Fixed in https://github.com/beacon-biosignals/EDF.jl/pull/26/commits/8e29613fe68cce4da7eaf83ef90ca6b10514ed97